test/marketpools.js

```diff
@@ -0,0 +1,895 @@
+/* eslint-disable */
```

Comment:
Can you see if using the fixtures in test/witnesses.js works for you? I've been meaning to refactor all tests to use it. Otherwise don't worry about it much, I'll probably do it all at once.

Reply:
I would like to refactor for this at a later time.

Reply:
sounds good.